Modules/_winapi.c

```diff
@@ -210,6 +209,9 @@ class HANDLE_return_converter(CReturnConverter):
         data.return_conversion.append(
             'return_value = HANDLE_TO_PYNUM(_return_value);\n')
 
+class DWORD_converter(unsigned_long_converter):
+    type = 'DWORD'
```

Comment:
You can use `DWORD(bitwise=True)` instead of `DWORD` in places where you want to keep the old behavior.

Or you can define non-strict by default completely compatible converter:
```python
class DWORD_converter(unsigned_long_converter):
    type = 'DWORD'

    def converter_init(self, *, bitwise: bool = True) -> None:
        super().converter_init(bitwise=bitwise)
```
and use `DWORD(bitwise=False)` if you need a strict conversion.